Sources/PackageLoading/ModuleMapGeneration.swift

```diff
@@ -106,7 +124,7 @@ extension ClangModule {
         case .directory(let path):
```

Comment:
If the style is a framework module, the module map header should be literally `framework module`